classroom.rb

```diff
@@ -0,0 +1,13 @@
+class ClassRoom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
```

Comment:
- [ ] In your Person class you should have `Getters and Setters for @rentals ` and @rentals should be initialized. 

`attr_accessor :name, :age, :rentals`


```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @rentals = []
end
```

- [ ] Please add a  `method for adding rentals`

```ruby
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```